src/axolotl/common/cli.py

```diff
@@ -21,13 +21,14 @@ class TrainerCliArgs:
     """
 
     debug: bool = field(default=False)
+    debug_text_only: bool = field(default=False)
+    debug_num_examples: int = field(default=5)
     inference: bool = field(default=False)
     merge_lora: bool = field(default=False)
     prepare_ds_only: bool = field(default=False)
     prompter: Optional[str] = field(default=None)
     shard: bool = field(default=False)
 
-
```

Comment:
you will probably need to add this back for `pre-commit` to pass

Reply:
Ah OK, done! Thanks